src/TLuaInterpreter.cpp

```diff
@@ -5110,7 +5101,8 @@ int TLuaInterpreter::playSoundFile( lua_State * L )
     {
         sound.replace('\\', "/");
     }
-    mudlet::self()->playSound( sound, soundVolume );
```

Comment:
I'd be tempted to make mudlet::playSound(...) return a boolean indicating whether that was successful; however given that we do not test for the sound file's actual existence (actually IIRC it could be a remote resource on the Internet so that is a whole different kettle of worms!) I'd fall back to issuing an unconditional:

    lua_pushboolean( L, true );
    return 1;

instead of nothing with the following:

    return 0;

because returning less arguments then the user has coded for will give them a _nil_ for the missing value(s). If the end user has got into the habit of checking for a _nil + error message_ for runtime value errors this means they will get a _nil + nil_ pair of values on __success__ (as far as __this__ function believes) instead of _true + nil_.  On the other hand I think rain is wet so who am I to say...